src/o365/graph/commands/teams/teams-clone.ts

```diff
@@ -0,0 +1,187 @@
+  }
+
+  public get description(): string {
+    return 'Creates a copy of a Microsoft Teams team';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.description = typeof args.options.description !== 'undefined';
+    telemetryProps.classification = typeof args.options.classification !== 'undefined';
+    telemetryProps.visibility = typeof args.options.visibility !== 'undefined';
+    return telemetryProps;
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    auth
+      .ensureAccessToken(auth.service.resource, cmd, this.debug)
+      .then((): request.RequestPromise => {
+
+        const requestOptions: any = {
+          url: `${auth.service.resource}/v1.0/teams/${encodeURIComponent(args.options.teamId)}/clone`,
+          headers: Utils.getRequestHeaders({
+            authorization: `Bearer ${auth.service.accessToken}`,
+            "content-type": "application/zip",
+            accept: 'application/json;odata.metadata=none'
+          }),
+          json: true,
+          body: {
+            displayName : args.options.displayName,
+            mailNickname : args.options.mailNickname,
+            partsToClone : args.options.partsToClone,
+            description : args.options.description || undefined,
+            classification : args.options.classification || undefined,
+            visibility : args.options.visibility || undefined,
+          }
+        };
+
+        if (this.debug) {
+          cmd.log('Executing web request...');
+          cmd.log(requestOptions);
+          cmd.log('');
+        }
+
+        return request.post(requestOptions);
+      })
+      .then((res: any): void => {
+        if (this.debug) {
+          cmd.log('Response:');
+          cmd.log(res);
+          cmd.log('');
+        }
+
+        if (this.verbose) {
+          cmd.log(vorpal.chalk.green('DONE'));
+        }
+
+        cb();
+      }, (res: any): void => this.handleRejectedODataJsonPromise(res, cmd, cb));
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-i, --teamId <teamId>',
+        description: 'The ID of the Microsoft Teams team to clone'
+      },
+      {
+        option: '-n, --displayName <displayName>',
+        description: 'The display name for the new Microsoft Teams Team'
+      },
+      {
+        option: '-m, --mailNickname <mailNickname>',
+        description: 'The mail alias for the new Microsoft Teams Team'
+      },
+      {
+        option: '-p --partsToClone <partsToClone>',
+        description: 'A comma-seperated list of the parts to clone. Allowed values are apps|channels|members|settings|tabs'
+      },
+      {
+        option: '-d --description [description]',
+        description: 'The description for the new Microsoft Teams Team. Will be left blank if not specified'
```

Comment:
Let us wait for Waldek's response.

Reply:
The description should reflect what the end-result is, so if the description is the same as displayName, we shouldn't say it will be blank.

Reply:
If we say it will be blank, will not the expectation to be to clone the team and see a blank desc field?

Reply:
Yes, on the other hand, if we leave it blank and the Graph always uses a default value, then we should align with what's Graph is actually. doing.

Reply:
@waldekmastykarz , my intention is to just remove the misleading message from the description. I already raised it as issue with the graph docs. This will not give at least wrong information for the user. What do you think?

Reply:
Good idea @VelinGeorgiev.

Reply:
@VelinGeorgiev, This is Done. Please review.

Reply:
@waldekmastykarz/@VelinGeorgiev., I have removed the misleading verbiage. Please review.